src/components/victory-group/victory-group.js

```diff
@@ -404,6 +411,7 @@ export default class VictoryGroup extends React.Component {
         theme: child.props.theme || props.theme,
         labelComponent: props.labelComponent || child.props.labelComponent,
         style,
+        domainPadding: {x: (props.width / noOfChildren) / 2},
```

Comment:
the default is 0, so the user would have to set an offset. but maybe that makes sense? or we could change the default offset.